programmers/K번째수.js

```diff
@@ -0,0 +1,10 @@
+function solution(array, commands) {
+    let newArray = [];
+    let answer = [];
```

Comment:
answer은 후에 '=' 연산자를 써서 값을 변화주는게 아니기 때문에 const가 맞음

```suggestion
    const answer = [];
    let newArray = [];
```